### ISSUE TITLE
Trained Model

### ISSUE THREAD

**Issue by Jie-su** [sherleens/WSCNet#3]
Hi, 

i Just train your script on Twitter I dataset , but the accuracy is not good. Could u provide trained models for all datasets?  Also, I can not find Flickr dataset download place. Could u also provide that

Very Thanks

**Comment by SomeO** [comment]
hi，sherleens，I am very interested in your work，And recently I read your paper and run your code, but the same problem I have encountered is no FI dataset，Could U provide the FI dataset download link，or any approach to get it？Thanks a lot

@sherleens

**Comment by iser97** [comment]
> Hi,
> 
> i Just train your script on Twitter I dataset , but the accuracy is not good. Could u provide trained models for all datasets? Also, I can not find Flickr dataset download place. Could u also provide that
> 
> Very Thanks

Hi, jie-su, I ran your code on the EmotionROI data set, and found that there are several differences with sherleens's code. I think the reason why your accuracy is not good is that you added the calculation of softmax in class ResNetWSL, and the nn.crossentropyloss used in calculating the loss contains the calculation of softmax, so when the softmax in class ResNetWSL is removed, the accuracy will be improved better. By the way, I can't find the FI dataset either.

**Comment by dzqdudley** [comment]
> > Hi,
> > i Just train your script on Twitter I dataset , but the accuracy is not good. Could u provide trained models for all datasets? Also, I can not find Flickr dataset download place. Could u also provide that
> > Very Thanks
> 
> Hi, jie-su, I ran your code on the EmotionROI data set, and found that there are several differences with sherleens's code. I think the reason why your accuracy is not good is that you added the calculation of softmax in class ResNetWSL, and the nn.crossentropyloss used in calculating the loss contains the calculation of softmax, so when the softmax in class ResNetWSL is removed, the accuracy will be improved better. By the way, I can't find the FI dataset either.

Hi iser97, I am trying to run the code onteh EmotionROI dataset as well, and I didn't change any structure or parameters of the script. But the accuracy is not good. In your code, do you change anything? Thanks!
